curriculum/challenges/english/25-front-end-development/lab-blog-post-card/66eaddd04a9e533fba689001.md

```diff
@@ -0,0 +1,239 @@
+---
+id: 66eaddd04a9e533fba689001
+title: Design a Blog Post Card
+challengeType: 14
+dashedName: "lab-blog-post-card"
+demoType: onClick
+---
+
+# --description--
+
+In this lab, you'll practice how to style backgrounds and borders.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should have a `div` with a class of `blog-post-card` to hold all your card elements.
+2. You should have an image with a valid `alt` text and the class of `post-img`. You can use `cover-photo.jpg` for the `src` attribute of your image.
+3. You should have a `div` with a class of `post-content`. Within the `.post-content` `div`:
```

Comment:
Seems a bit better this way regarding readability.
Not something necessary though.

```suggestion
3. You should have a `div` with a class of `post-content`. 
4. Within the `.post-content` `div`:
```